builder/docker/doc.go

```diff
@@ -0,0 +1,7 @@
+/*
+Package docker provides Docker utlities.
+
+Some of the utilities rely upon the presence of the command line client for
+Docker. Others use the Docker API.
+*/
+package docker
```

Comment:
I like this convention of putting the package comment in `doc.go`. I tend to put it in `main.go` or any random source file, so that readers have to search for it. :disappointed: